### PR TITLE
Changelog sprint update

### PR DESCRIPTION
**Pull request by chibenwa**
Merge this at the end of the Sprint.

Should we mention JIRA issue numbers?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## [Unreleased]
 ### Changed
 - Multiple libraries updates
+- Drop partially implemented Kafka distributed events
```
Comment (rouazana) [inline CHANGELOG.md]: should be in a ### Removed section

**Comment by rouazana** [comment]
BTW I like this process (opening a PR a the start of the sprint and complete it as and when).

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- `${project.groupId}` in dependencyManagement is harmful
```
Comment (mbaechler) [inline CHANGELOG.md]: doesn't worth sharing IMO

Comment (chibenwa) [reply CHANGELOG.md]: It's remove :-)

**Comment by rouazana** [comment]
see #1670

**Pull request closed without merging**